src/ngx_http_lua_worker.c

```diff
@@ -190,4 +190,33 @@ ngx_http_lua_ffi_process_signal_graceful_exit(void)
 {
     ngx_quit = 1;
 }
+
+
+int
+ngx_http_lua_ffi_enable_sa_restart(int signum, int enabled)
+{
+    struct sigaction        act;
```

Comment:
I'm afraid this breaks compatibility with systems like Windows. Better do a feature test to make sure we have these signal stuffs?

Reply:
Both comments addressed. Is it worth starting to invest time in tests and documentation for this?